### PR TITLE
feat: dark mode & light mode

### PR DESCRIPTION
**Pull request by CoderMungan**
Ekran Resmi 2024-01-29 00 03 41

### CODE REVIEW

File: archaeologyMain/templates/Components/_navbar.html [hunk 1]
```diff
@@ -115,10 +114,13 @@
                         </li>
                     </ul>
                 </li>
-
+                <div class="ms-1">
+                    <button class="btn btn-outline-success" id="darkModeToggle"><i class="fa-solid fa-moon"></i> / <i
+                            class="fa-solid fa-sun"></i></button>
+                </div>
```
Comment (mciray) [inline archaeologyMain/templates/Components/_navbar.html:120]: gj

**Review by mciray — APPROVED** [review]
look good to me!

**Review by mciray — APPROVED** [review]
lgtm

**Review by AysKrimn — APPROVED** [review]

**Comment by AysKrimn** [comment]
great.

**Pull request merged**